core/src/rpc_pubsub.rs

```diff
@@ -582,7 +582,7 @@ mod tests {
                        "lamports": 51,
                        "data": bs58::encode(expected_data).into_string(),
                        "executable": false,
-                       "rentEpoch": 1,
+                       "rentEpoch": 0,
```

Comment:
@CriesofCarrots @jstarry Strictly speaking, we're breaking some semantics (=~ compatibility) of our rpc api a bit here according to this new rule for `rentEpoch`: https://github.com/solana-labs/solana/pull/11349/files#r469691300.

It's an incompatible change, but the change is soooo subtle, I think. So, I believe there should be virtually no one relying on `rentEpoch` being the current epoch or the next in any-conceivable way for the bizarre usecases.

Reply:
Or, if you have something to come to mind in terms of various partner integartions, please let me know. :)

Reply:
@ryoqun agreed, very unlikely this will cause any issues. I highly doubt anyone is using `rentEpoch` right now

Reply:
I don't know of any integrations using `rentEpoch`; I'm comfortable with this breaking change.